source/gui/__init__.py

```diff
@@ -656,20 +656,35 @@ def __init__(self, parent):
 			# Translators: An option in the combo box to choose exit action.
 			_("Restart")
 		]
-		# Windows Store version of NVDA does not support add-ons yet.
-		if not config.isAppX:
-			# Translators: An option in the combo box to choose exit action.
-			self.actions.append(_("Restart with add-ons disabled"))
-		# Translators: An option in the combo box to choose exit action.
-		self.actions.append(_("Restart with debug logging enabled"))
 		if updateCheck and updateCheck.isPendingUpdate():
 			# Translators: An option in the combo box to choose exit action.
 			self.actions.append(_("Install pending update"))
 		self.actionsList = contentSizerHelper.addLabeledControl(labelText, wx.Choice, choices=self.actions)
 		self.actionsList.SetSelection(0)
 
+		# Translators: Label for restart options group.
+		restartOptionsLabel = _("Restart options.")
+		restartGroupSizer = wx.StaticBoxSizer(wx.VERTICAL, self, restartOptionsLabel)
+		self.restartGroupBox = restartGroupSizer.GetStaticBox()
+		self.restartGroup = guiHelper.BoxSizerHelper(self, sizer=restartGroupSizer)
+
+		self.disableAddonsCB = self.restartGroup.addItem(
+			# Translators: An option in the group to choose restart action.

```

Comment:
IMO "options" fits better than "action":

```suggestion
			# Translators: An option in the group to choose restart options.
```